test/integration/scripting_spec.js

```diff
@@ -804,6 +804,10 @@ describe("Interaction", () => {
             window.document.querySelector(selector).scrollIntoView();
           }, getSelector("171R"));
 
+          await page.waitForSelector('[data-page-number="4"][class="page"]', {
```

Comment:
I'm probably missing something obvious here, but aren't the fields we're filling out below actually on the *second* page of the document?

Reply:
Yes that's correct but some fields are updated on page 4.
Here the goal is to be sure that page 4 is here before trying to scroll it into the view.